docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
+NVIDIA GPU. This is achieved by configuring the underlying MicroK8s, that the
+DSS is relying for running the containerised workloads.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+* Have run the :ref:`tutorial` tutorial
+
+Install the NVIDIA Operator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To ensure the DSS can utilise the NVIDIA GPUs:
+
+1. The host will need to have the NVIDIA drivers installed
```

Comment:
These two points seems to be the actual prerequisites to enable GPUs. Then, we can move them to the previous section.

Reply:
Good point! I'll move them at the top